dataprofiler/profilers/profiler_options.py

```diff
@@ -376,6 +376,42 @@ def _validate_helper(self, variable_path='IntOptions'):
         return super()._validate_helper(variable_path) 
 
 
+class PrecisionOptions(BooleanOption):
```

Comment:
Not for this PR, but we may want to look into consolidating naming in `profiler_options` currently base, boolean, and histogram are in singular form (i.e. BooleanOption) while all others are plural (FloatOptions). This isn't a super big deal but just something to note.